idmega2000/trabajo
# Set up express server

#### What does this PR do?
It set's up the node js express server
#### Description of Task to be completed?
Set's up express server

#### How should this be manually tested?
fetch the branch and cd into the app
run `npm run dev`, the app should start on the specified port
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
n/a
#### Screenshots (if appropriate)
n/a
